libglusterfs/src/iobuf.c

```diff
@@ -459,12 +459,12 @@ __iobuf_get(struct iobuf_pool *iobuf_pool, const size_t page_size,
     if (!iobuf_arena)
         return NULL;
 
-    list_for_each_entry(iobuf, &iobuf_arena->passive.list, list) break;
+    list_for_each_entry(iobuf, &iobuf_arena->passive_list, list) break;
```

Comment:
Maybe this would be simpler and cleaner:

```suggestion
    iobuf = list_first_entry(&iobuf_arena->passive_list, struct iobuf, list);
```

Reply:
For sure, I just did not want to introduce additional unrelated changes. If I had, I would have removed this passive and active lists altogether and use large bitmaps (say, 2-4 of 64bit each) to check for available iobufs ;-)